prime-angular-frontend/src/app/shared/components/contact-information-form/contact-information-form.component.html

```diff
@@ -44,12 +44,15 @@
     <app-page-subheader>
       <ng-container appPageSubheaderTitle>Notification Information</ng-container>
       <ng-container appPageSubheaderSummary>
-        Provide an email address 
-        <ng-container *ngIf="showSmsPhone">
-          and phone number (optional) 
-        </ng-container>
-        that may be used for sending notifications from the PRIME
-        system.
+        <div *ngIf="!forPhsa else PhsaSummary">
```

Comment:
I'm surprised this works unless there is an update to `*ngIf` structural directive that I'm not aware of but typically there's a semicolon between the statements `*ngIf="!forPhsa; else PhsaSummary"`